examples/a-regression/script.jl

```diff
@@ -154,7 +159,7 @@ scatter(
     markeralpha=0.1,
     label="Training Data",
 )
-plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4)
+DisplayAs.PNG(plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4))
```

Comment:
Was just wondering if you tried something like in https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@example-block, e.g.
```suggestion
plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4)
DisplayAs.PNG(ans) # hide
```
or
```suggestion
plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4)
savefig("posterior.png"); nothing # hide
```

Reply:
Oh the first one looks good! Thanks! I don't know about the second one